src/Infrastructure/Form/Regulation/LocationFormType.php

```diff
@@ -68,14 +102,52 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 options: [
                     'label' => 'common.form.validate',
                 ],
-            )
-        ;
+            );
+    }
+
+    private function getRoadTypeOptions(): array
+    {
+        $choices = [];
+
+        foreach (RoadTypeEnum::cases() as $case) {
+            $choices[sprintf('regulation.location.road.type.%s', $case->value)] = $case->value;
+        }
+
+        return [
+            'choices' => array_merge(
+                ['regulation.location.type.placeholder' => ''],
+                $choices,
+            ),
+            'label' => 'regulation.location.type',
+        ];
+    }
+
+    private function getAdministratorOptions(array $administrators): array
+    {
+        $choices = [];
+
+        foreach ($administrators as $value) {
+            $choices[$value] = $value;
+        }
+
+        return [
+            'label' => 'regulation.location.administrator',
```

Comment:
Ici utiliser cette même astuce et retirer l'étoile dans la traduction

```suggestion
            'label' => 'regulation.location.administrator',
            'label_attr' => [
                'class' => 'required',
            ],
```